src/briefcase/integrations/android_sdk.py

```diff
@@ -5,11 +5,36 @@
 
 from requests import exceptions as requests_exceptions
 
-from briefcase.exceptions import BriefcaseCommandError, NetworkFailure
+from briefcase.console import select_option
+from briefcase.exceptions import (
+    BriefcaseCommandError,
+    InvalidDeviceError,
+    NetworkFailure,
+)
 
 DEVICE_NOT_FOUND = re.compile(r"^error: device '[^']*' not found")
 
 
+class AndroidDeviceNotAuthorized(BriefcaseCommandError):
+    def __init__(self, device):
+        self.device = device
+        super().__init__(
+            """
+The device you have selected ({device}) has not had Developer Options
```

Comment:
Pedantic: Google doesn't capitalize Developer Options like this.

Useful feedback: it's **USB debugging** that the user needs to enable. It's good IMHO to change the string to say "USB debugging".

Reply:
There's 2 actions required (the 7 presses on build number dance, and then enabling USB debugging) - so it's worth mentioning both. The capitalization note is a good point, though.